sphinx/src/orga/guidelines.rst

```diff
@@ -192,7 +192,7 @@ Generalities
 
 Required:
 
-#. All interactions in the review adhere to our :ref:`code_of_conduct`.
+#. All interactions in the review adhere to our `code_of_conduct`.
```

Comment:
This should link somewhere. Either https://github.com/conda-forge/conda-forge.github.io/blob/main/README.md#code-of-conduct or `/community/governance.md#code-of-conduct`.